### PR TITLE
Homepage 

### PR DESCRIPTION
**Pull request by shraddha15-creator**
Added the homepage 

HOMEPAGE

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
+import { Route, Router, Routes } from "react-router-dom";
```
Comment (devdeadviz) [inline src/App.js]: You don't need to import `Router` from the `react-router-dom` package.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 import "./App.css";
+import Navbar from "./components/Navbar/Navbar";
 import logo from "./logo.png";
```
Comment (devdeadviz) [inline src/App.js]: Remove unused code.

File: src/App.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+import Home from "./pages/Home/Home";
+import { BrowserRouter } from "react-router-dom";
+import Footer from "./components/Footer/Footer";
```
Comment (devdeadviz) [inline src/App.js]: You can export all your components from a single `index.js` file, from that way you can import all your components from a single file.

File: src/components/Navbar/style.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    padding: .5rem;
+}
+
+.right-nav a i {
+    color: white;
+}
+
+.brand-name {
+    color: var(--white);
+}
```
Comment (devdeadviz) [inline src/components/Navbar/style.css:16]: Use more descriptive color names.

File: src/pages/Home/Home.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    border: 2px solid var(--black);
+    background-color: var(--white);
+}
+
+.shop-now-btn a:hover {
+    color: var(--black);
+}
+
+.section-name {
+    color: var(--white);
```
Comment (devdeadviz) [inline src/pages/Home/Home.css:20]: Use more descriptive color names.

File: src/pages/Home/Home.css [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    font-weight: 600;
+}
+
+.txt-overlay {
+    position: absolute;
+    top: 10%;
+    left: 20%;
+    text-align: center;
+    color: var(--white);
+    /* text-shadow: 2px 2px 3px var(--white); */
```
Comment (devdeadviz) [inline src/pages/Home/Home.css]: Remove commented codes.

**Review by devdeadviz — COMMENTED** [review]
Added some suggestions.

File: src/pages/Home/Home.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+							</button>
+						</a>
+					</div>
+				</div>
+
+				{/* <!-- Shop by category --> */}
+				<h3 className="section-name">Shop by Category</h3>
+				<div className="brands-container">
+					{guitarCategories &&
+						guitarCategories.map((item) => {
```
Comment (narharikale) [inline src/pages/Home/Home.js]: we can destructure category object here, it will more clear and concise

**Review by narharikale — APPROVED** [review]
gave some suggestion and i think there is some problem with your code formatter ,  it will be easier to review code

**Comment by shraddha15-creator** [comment]
> gave some suggestion and i think there is some problem with your code formatter , it will be easier to review code

Im using Prettier code formatter, don't know why still its giving problem. Thank you.

**Review by devdeadviz — APPROVED** [review]
Good to go!

**Pull request merged**